### PR TITLE
improve ln preformance when torch backend

### PR DESCRIPTION
**Pull request by pass-lin**
from https://github.com/keras-team/keras/issues/21313
This is the first time we have submitted a performance optimization for torch.
In the future, we will gradually replace the pure ops implementation with the cudnn implementation provided in the torch to improve the performance of the Layer.

### CODE REVIEW

File: keras/src/layers/normalization/layer_normalization.py [hunk 1]
```diff
@@ -214,6 +232,16 @@ def _broadcast(v):
...
                 inputs * inv * ops.cast(_broadcast(self.gamma), inputs.dtype)
             )
+        elif backend.config.backend() == "torch" and is_continue_axis(
+            self.axis
+        ):
+            # when using torch backend,use kernel to improve performance
+            import torch.nn.functional as F
+
+            normalized_shape = tuple([input_shape[dim] for dim in self.axis])
+            outputs = F.layer_norm(
```
[the remark this answers is not in the document]
Comment (pass-lin) [reply keras/src/layers/normalization/layer_normalization.py:242]: > We should not create backend-specific special-case at the layer level. The way we'd normally want to structure something like this is by introducing a backend op in all backends and having the perf optimization at that level`layer_normalization`

If it's reasonable, I'll close this PR. And then submit it again at the OP level.
